clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -1055,4 +1074,43 @@ private String getAsgName(com.netflix.spinnaker.clouddriver.titus.client.model.J
     }
     return asgName;
   }
+
+  /**
+   * For each server group with more than 1 job ID, log out all all the duplicate IDs, and log the
+   * final cached job ID too.
+   */
+  private void logDuplicateServerGroups(
+      Map<String, List<String>> seenServerGroupByJobIds, Map<String, CacheData> serverGroupCache) {
+    seenServerGroupByJobIds.entrySet().stream()
+        .filter(it -> it.getValue().size() > 1)
+        .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue))
```

Comment:
I think you could skip the collect here since you are already dealing with a stream of entries of key -> value

Reply:
Oh yes, of course.